### PR TITLE
DOC-3426: Added missing flag to commands. Adding sentence explaining flag.

### PR DESCRIPTION
**Pull request by taylorharvey-r3**

### CODE REVIEW

File: content/en/platform/corda/5.0-dev-preview-1/tutorials/building-cordapp/c5-basic-cordapp-running.md [hunk 1]
```diff
@@ -72,37 +74,35 @@ nodes:
...
 
-7. Start Docker containers by running the following command:
-
-`docker-compose -f docker-compose.yaml up -d`
+```
+corda-cli network deploy -n missionmars-network -f missionmars.yaml | docker-compose -f - up -d
+```
 
-This step will take some time to complete.
+Here the `-f` flag ensures that the network files have the same name as the network itself. This step will take some time to complete.
```
Comment (peterli-r3) [inline content/en/platform/corda/5.0-dev-preview-1/tutorials/building-cordapp/c5-basic-cordapp-running.md:83]: ah no. Just remove what you added there. LOL

Comment (benmcmahon100) [inline content/en/platform/corda/5.0-dev-preview-1/tutorials/building-cordapp/c5-basic-cordapp-running.md:83]:
```suggestion
Here the `-f` flag allows us to specify the location of the network definition file. This step will take some time to complete.
```

File: content/en/platform/corda/5.0-dev-preview-1/tutorials/run-demo-cordapp.md [hunk 2]
```diff
@@ -87,7 +87,11 @@ The project containing the sample CorDapp opens.
 This command builds the CorDapp package (`.cpk`) files. Most CorDapps have two files: a `contracts` file and a `workflows` file. The command also builds the CorDapp package bundle (`.cpb`).
 
 4. Deploy the network using `corda-cli` and `docker-compose`:
-  `corda-cli network deploy -n solar-system | docker-compose -f - up`
+```
+corda-cli network deploy -n solar-system -f solar-system.yaml | docker-compose -f - up -d
+```
+
+Here the `-f` flag ensures that the network files have the same name as the network itself.
```
Comment (benmcmahon100) [inline content/en/platform/corda/5.0-dev-preview-1/tutorials/run-demo-cordapp.md:94]: We could omit the -f from the previous command as the network and network definition yaml have the same name, not sure if we need to mention it though, it should more be something documented on the command guide page where we explain what all the flags do.

**Review by benmcmahon100 — DISMISSED** [review]
Perfect, thanks @taylorharvey-r3

**Pull request merged**